frontend/api/obs-frontend-api.h

```diff
@@ -80,6 +83,18 @@ static inline void obs_frontend_source_list_free(struct obs_frontend_source_list
 	da_free(source_list->sources);
 }
 
+struct obs_frontend_canvas_list {
+	DARRAY(obs_canvas_t *) canvases;
+};
+
+static inline void obs_frontend_canvas_list_free(struct obs_frontend_canvas_list *canvas_list)
+{
+	size_t num = canvas_list->canvases.num;
+	for (size_t i = 0; i < num; i++)
+		obs_canvas_release(canvas_list->canvases.array[i]);
+	da_free(canvas_list->canvases);
```

Comment:
Add a blank line before the `da_free` to separate it from the `if`, or enclose the `if` body in curly braces.